pandas/plotting/_matplotlib/core.py

```diff
@@ -440,7 +441,9 @@ def _compute_plot_data(self):
         if is_empty:
             raise TypeError("no numeric data to plot")
 
-        self.data = numeric_data.apply(self._convert_to_ndarray)
+        self.data = numeric_data.apply(
```

Comment:
can you just change _convert_to_ndarray to handle this case? (and mention in the doc-string)